receiver/postgresqlreceiver/README.md

```diff
@@ -95,6 +97,12 @@ The following options are available:
 - `max_rows_per_query`: (optional, default=1000) The max number of rows would return from the query 
 against `pg_stat_statements`.
 - `top_n_query`: (optional, default=1000) The maximum number of active queries to report (to the next consumer) in a single run.
+- `max_explain_each_interval`: (optional, default=1000). The maximum number of explain query to be sent in each scrape interval. The top query 
```

Comment:
I feel that this flag seems not that useful? May we can introduce a flag to disable the reporting query plan behavior? I feel that it would be weird that users need to change two configs if they really want to increase the `top_n_query` - Just a though, may not be correct.

Reply:
my intention was to decrease the duration of the `top query collection`. otherwise, it will send the `explain query` for all the not cached yet query. with this config, user can control how many number of `explain query` to be executed for one `top query collection`. to disable it, just set it to `0` should be enough

Reply:
sounds good